parcellab/common/templates/_container.tpl

```diff
@@ -0,0 +1,82 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+  Keys and quoted values generated from a given dict:
+  {{ include "common.container" (
+    dict
+      "container" "The specific pod configuration"
+      "type" "The tye of container to define /optional (defaults to 'service')"
```

Comment:
From what I see we are missing `commonRefName`, `name`, `image`, `containerEnv`, `volumes`, `datadog`, etc...

It is important to document the inputs because we do not have much help with this markup language that will improve the readibility of the code (I do not know what are the "function arguments").

Reply:
I see the entire spec in the other file:

```
(dict "volumes" $podVolumes "containerEnv" $containerEnv "datadog" $datadog "commonExternalSecret" $commonExternalSecret "commonConfig" $commonConfig "commonRefName" $fullname)) 
```

A small description (or at least defining it here) helps a lot when reading the template